homeassistant/components/decora_wifi/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Setup Decora Wifi",
+        "description": "Your my.leviton.com credentials",
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
```

Comment:
Instead of setting such description, add a `data_description` field to add context to `username` and `password`